### PR TITLE
Additional Access Re-evaluation

### PR DESCRIPTION
**Pull request by Ikarrus**
Additional access hasn't been used nor even looked at since errorage first coded it years ago. Departments and accesses were added, removed, and moved around. Basically, it's outdated.

This PR re-evaluates what additional access includes. The original idea was to give jobs most access to other workspaces in their department, so I'm going to keep with that direction. For example, a medical doctor would be able to work anywhere in Medbay (Except CMO's office).

It also turns out sec/assmaint is already included in additional access so I don't need to include them in the automatic toggle.

### CODE REVIEW

File: code/game/jobs/job/captain.dm [hunk 1]
```diff
@@ -70,7 +70,7 @@ Head of Personnel
 	default_pda = /obj/item/device/pda/heads/hop
 	default_headset = /obj/item/device/radio/headset/heads/hop
 
-	access = list(access_security, access_sec_doors, access_brig, access_court, access_forensics_lockers, access_weapons,
```
Comment (Iamgoofball) [inline code/game/jobs/job/captain.dm]: hold up here what's going on with captain losing access

**Comment by Iamgoofball** [comment]
i can't read/10

merge post-haste

**Comment by Deantwo** [comment]
Seems like your PR is reverting all of #7244.

**Comment by Ikarrus** [comment]
Sec/Ass maint is already covered by jobs_have_minimal_access 0, so I just took out some redundancy and clarified the comments.

**Comment by Deantwo** [comment]
Ah ok. Just wanted to ask, but kinda figured you knew what you were when I remembered #7244 was yours too.
Just wanted to be 100% sure.

**Comment by Ikarrus** [comment]
No worries, mate.

**Comment by AlexCrimson** [comment]
So basically, when the server is under 20 players, department job get more access? Like Doctors can walk into Chemistry, or Scientists can do Robotics?

**Comment by Ikarrus** [comment]
Yes. You should still have your job as a priority , though, but you're able to cover for missing co-workers during low pop.

**Comment by as334** [comment]
Holy shit geneticist's basically become mini-heads in terms of access with this.

**Pull request merged**